index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <header>
+        <span class="logo-placeholder">
+            Kenny Salazar
+        </span>
+        <div class="menu-icon">
+        </div>
```

Comment:
The hamburger button in our mobile design should be a nav tag to meet the project requirements. Kindly make the necessary changes 🙌